client/src/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+
+class App extends React.Component {
```

Comment:
I know you are only on the planning stage. Remember to use stateless component if you only render and return stateless element.    😃

Reply:
Thanks, I need to remember to use those more often!